### DISCUSSION TITLE
Add external analytics

### DISCUSSION THREAD

**Discussion by alexandremjacques** [welpo/tabi#356] [Q&A]
Hi.

Is it possible to add an external analytics tool without having to change the `base.html` file? 

I need to "inject" something like in the `<head>` tag:
```
<script async src="https://cdn.seline.so/seline.js"></script>
```

What would be the best approach?

TIA

**Comment by welpo** [comment]
Hi Alexandre!

Right now it's not possible to load custom scripts without modifying the theme files. It would not be hard to add the feature, though.

For this particular use case, since Seline appears to be privacy-respecting, I would be happy to add it as a supported analytics service, with its own configuration options and automatic [CSP](https://welpo.github.io/tabi/blog/security/) settings.

Would you be interested in creating a PR for either modification (load user-specified scripts / support Seline as analytics)?
